rust/xain-grpc/certs/ca-key.pem

```diff
@@ -0,0 +1,30 @@
+-----BEGIN ENCRYPTED PRIVATE KEY-----
```

Comment:
Can't we generate these locally instead of commiting them? E.g. generate if not present. I think this would avoid potential mishaps in the long run.

Reply:
We could do so, though I don't see a huge potential for this, as Rust code is usually shipped out of the target directory.

We could move them to the test directory first.

I'll write a script to generate them, also to make sure we can regenerate them with the same settings.